spirit_simulator/a1_description/sdf_mesh/a1_new.sdf

```diff
@@ -0,0 +1,1128 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should only have one sdf/urdf for our repo, no need to keep the outdated one. Those files should be named <robot_name>.urdf/sdf so that they work with our launchfiles.

Reply:
Ah, we might ignore this pull request at this point. I submit another one with both a1 description added and other changes included to make a1 work.